test/src/api/commands/web-element/assert/testElementAssertions.js

```diff
@@ -213,7 +213,8 @@ describe('element() assert commands', function () {
     // }
   });
 
-  it('test .element() hasAttribute assert', async function() {
+  // TODO: enable once below assertion is fixed
+  xit('test .element() hasAttribute assert', async function() {
```

Comment:
Figure out why this is failing. There _could_ be a mock that is not being cleaned up that is causing this to fail

Reply:
Not failing anymore, I just had to pull in main.

Reply:
updated